test/integration/render-tests/collator/default/style.json

```diff
@@ -0,0 +1,225 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "height": 512,
```

Comment:
Can this work at a smaller size (w/ a corresponding smaller font size)?  Or does that make the image illegible?

Reply:
Yeah I brought it down to 256x256 at 11 point font, still easy to read.